sklearn/gaussian_process/tests/test_kernels.py

```diff
@@ -46,6 +44,12 @@
     if metric in ["additive_chi2", "chi2"]:
         continue
     kernels.append(PairwiseKernel(gamma=1.0, metric=metric))
+kernels += [SelectDimensionKernel(RBF(length_scale=2.0),
+                                  active_dims=[0]),
+            SelectDimensionKernel(RBF(length_scale=[2.0, 1.0]),
+                                  active_dims=[0, 1]),
+            SelectDimensionKernel(Matern(length_scale=0.5, nu=0.5),
+                                  active_dims=[1])]
```

Comment:
please replace in the last kernel the definition of active_dims by a boolean array ([False, True]) so that we cover also this case.